website/static/js/fangorn.js

```diff
@@ -1372,7 +1372,7 @@ var FGButton = {
         }
         return m('div', opts, [
             m('i', {className: iconCSS}),
-            m('span', children)
+            (children? m('span', children): m(''))
```

Comment:
`m('')` - could this be replaced by an empty string?

Further, perhaps something like this could be used:

```javascript
    var childOptions = [m('i', {className: iconCSS})];
    if (children) {
        childOptions.push(m('span', children));
    }
    return m('div', opts, childOptions);
```

My apologies if the above is invalid, I'm not terribly familiar with Mithril yet, and I'm writing it in a GitHub comment pane. :cry: